### PR TITLE
doc: Update code docs with type

### PR DESCRIPTION
**Pull request by ashcrow**

### CODE REVIEW

File: molecule/ansible_playbook.py [hunk 1]
```diff
@@ -17,6 +17,9 @@
 #  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
 #  FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
 #  DEALINGS IN THE SOFTWARE.
+"""
+ansible playbook runner.
```
Comment (retr0h) [inline molecule/ansible_playbook.py]: Capitalize Ansible?

File: molecule/cli.py [hunk 2]
```diff
@@ -17,10 +17,14 @@
 #  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
 #  FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
 #  DEALINGS IN THE SOFTWARE.
+"""
+command line interface.
```
Comment (retr0h) [inline molecule/cli.py]: Capitalize command.

**Comment by retr0h** [comment]
Might be of interest to look at v2 branch of molecule, which is the future direction of the project.

**Comment by ashcrow** [comment]
@retr0h doh! I didn't notice that it was a separate branch.

**Pull request closed without merging**